packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -223,10 +223,13 @@ class TextPainter {
   /// parameters, lays it out with the supplied [minWidth] and [maxWidth], and
   /// returns its [TextPainter.width] making sure to dispose the underlying
   /// resources.
+  ///
+  /// Unlike the [TextPainter] constructor, the [textDirection] parameter is
```

Comment:
The grammar in this sentence is dubious (it implies "the constructor is not required"). In general I'm not sure as written that this paragraph really helps the reader understand what's happening.

I would instead say something like "The [text] and [textDirection] are required as they have no reasonable defaults." or even maybe just not say anything at all.

Reply:
Removed.